docs/markdown/Reference-tables.md

```diff
@@ -38,6 +38,7 @@ These are return values of the `get_id` (Compiler family) and
 | sun       | Sun Fortran compiler             |                 |
 | c2000     | Texas Instruments C/C++ Compiler (C2000) |                 |
 | ti        | Texas Instruments C/C++ Compiler |                 |
+| c6x       | Texas Instruments C/C++ Compiler (C6000) |                 |
```

Comment:
Why don't these follow what was done for `c2000` DSPs? In particular, below the `ar6x` linker description says that it is used only with the `C6000` DSPs, but the `cpu` is defined as `tidsp`. Either the `ar6x` works with more DPS classes, or the addition should be revisited to follow what was done for the `c2000` DSPs, using uniform names for the compilers and linkers.